spec/features/changing_answers_spec.rb

```diff
@@ -23,6 +23,31 @@
     expect(page).to have_content("Check your answers before sending your application")
   end
 
+  context "when changing the year they qualified" do
```

Comment:
This feature spec file takes over 40 seconds to run on my machine, mostly because of the way features are being tested with multiple scenarios: each scenario will execute `answer_all_student_loans_claim_questions`, which clicks through every single page of the journey. As a separate issue we should definitely to rework these specs so that there are fewer scenarios, but for now, can we try and write new specs with as few scenarios as possible? In this case, I think this feature can be tested with a single `"changes their QTS year"` scenario containing multiple expectations.